packages/style-kit/src/_inputs.scss

```
@@ -20,46 +32,90 @@
   margin-bottom: -$grid;
 }
 
-@mixin search-input-container($theme) {
-  padding: 0px $grid;
-  border-radius: $border-radius-large;
-  border: 1px solid map-get($theme, 'light');
-  background-color: map-get($theme, 'bgShade');
-  display: flex;
-  align-items: center;
-  max-width: 360px;
+@mixin attachment($theme) {
+  @include input($theme);
+  background-color: transparent;
+  border-width: 1.5px;
+  display: inline-flex;
+  flex-direction: column;
+  width: 60%;
+  padding: $grid / 2 $grid * 8 $grid / 2 $grid / 2;
```

Maybe add some parens for readability:

```suggestion
  padding: ($grid / 2) ($grid * 8) ($grid / 2) ($grid / 2);
```